frontend/apps/filemanager/filemanager.lua

```diff
@@ -1052,6 +1052,23 @@ function FileManager:moveFile(from, to)
     return BaseUtil.execute(self.mv_bin, from, to) == 0
 end
 
+--[[
+A shortcut to execute cp command (self.cp_bin) with from and to as parameters.
+Returns a boolean value to indicate the result of cp command.
+--]]
```

Comment:
I think that should be just like this, unless ldoc is happy to ignore it.
```suggestion
]]
```